### DISCUSSION TITLE
请问如何更改章节名的格式，比如把“1 绪论”改成第1章 绪论

### DISCUSSION THREAD

**Discussion by zilre24** [TheNetAdmin/zjuthesis#401] [General]
请问如何更改章节名的格式，比如把“1 绪论”改成“第1章 绪论”

**Comment by TheNetAdmin** [comment]
修改这里，参考ctex宏包的文档，文档内应该有具体的例子

https://github.com/TheNetAdmin/zjuthesis/blob/0464c71837748cc4ca0ef57aeb5d0bee4b75ef2b/config/format/general/heading.tex#L11-L27

**Comment by zilre24** [comment]
好的，感谢感谢~
        chapter =
        {
            format=\raggedright\zihao{-3}\bfseries,
            name={第,章},
            number=\arabic{chapter},
            beforeskip=12pt,
            afterskip=6pt
        }

**Comment by Chenyanlu99** [comment]
以下代码可以使章节名居中并将阿拉伯数字改为中文数字：
chapter =
{
    format=\centering\zihao{-3}\bfseries, % 将标题居中
    name={第,章},
    number=\chinese{chapter}, % 使用中文数字
    beforeskip=12pt,
    afterskip=6pt
},

**Comment by Hello-JoshuaLee** [comment]
如果是研究生的论文，改动56行开始的chapter为：
chapter =
        {
            format=\centering\zihao{-3}\bfseries,
            name={第,章},
            number=\arabic{chapter},
            beforeskip=12pt,
            afterskip=6pt
        },
